### ISSUE TITLE
Hide character feature images from screen readers

### ISSUE THREAD

**Issue by moT01** [freeCodeCamp/freeCodeCamp#52554]
> The first thing I notice is that I can hear a bunch of "graphic character brows" "graphic character brows" etc.

_Originally posted by @naomi-lgbt in https://github.com/freeCodeCamp/freeCodeCamp/issues/52543#issuecomment-1858035717_

We should hide the character images with `aria-hidden='true'` - we can probably hide all the images, except for maybe the base character.

**Comment by bbsmooth** [comment]
Yes, definitely, these images should have empty alt text!

**Comment by igoroganesian** [comment]
Alt text emptied 🫡
